packages/hw-app-xrp/README.md

```diff
@@ -0,0 +1,9 @@
+<img src="https://user-images.githubusercontent.com/211411/34776833-6f1ef4da-f618-11e7-8b13-f0697901d6a8.png" height="100" />
+
+## @ledgerhq/hw-app-eth
```

Comment:
please update the name of the lib. (that kinda sucks we will try to have template in the future)